packages/vite/src/node/server/ws.ts

```diff
@@ -108,7 +108,7 @@ export function createWebSocketServer(
     const port = hmrPort || 24678
     const host = (hmr && hmr.host) || undefined
     if (httpsOptions) {
-      // if we're serving the middlewares over https, the ws library doesn't support automatically creating an https server, so we need to do it ourselves
+      // if we're serving the middlewares over https, the ws library doesn't support automatically creating a https server, so we need to do it ourselves
```

Comment:
```suggestion
      // if we're serving the middlewares over https, the ws library doesn't support automatically creating an https server, so we need to do it ourselves
```

The original text is correct. Use `an` before an unsounded `h`.